src/domain_bridge/domain_bridge.cpp

```diff
@@ -294,9 +294,20 @@ class DomainBridgeImpl
         topic_options.remap_name(), options_.name(), "/");
     }
 
+    // Prevent using reversed flag and bidirectional flag at the same time
```

Comment:
Though setting bidirectional to `true` subverts the reversed option, is there actually an issue if we create a bridge with both options set? I'm just wondering if we actually need to print an error here.

Reply:
The way that I had written this originally, setting both options would cause the topic to bridge in the same direction twice (which is what I was trying to guard against). But you're right, in its current state there isn't really a problem using both options. I've removed this warning in 7b06684.